android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/Cloudlet.java

```diff
@@ -42,8 +42,6 @@
 import fr.bmartel.speedtest.inter.ISpeedTestListener;
 import fr.bmartel.speedtest.model.SpeedTestError;
 
-import static com.mobiledgex.sdkdemo.CloudletListHolder.DownloadTestType.staticFile;
-
 public class Cloudlet implements Serializable {
     private static final String TAG = "Cloudlet";
     public static final int BYTES_TO_MBYTES = 1024*1024;

```

Comment:
Actually should be BITS_TO_MBITS since it is used to convert in bits/second to be displayed as Mbits/sec. Why 1024*1024? Well, I'm glad someone finally caught that. I was thinking about base 2 for like for drive sizes, but it seems 1000*1000 is more commonly used for data rates. I'll do more research, but it looks like I'll be changing it.

Reply:
*cough*: https://en.wikipedia.org/wiki/Mebibyte

Reply:
Aha i see. Can we use BYTES_TO_MIBYTES then?

Reply:
It seems base 10 is the standard for measuring transfer rates. Good definition of Mbps here: https://single.speedtest.net/help#glossary 1000*1000. That's my final answer.